cmake-modules/AddTestCMocka.cmake

```diff
@@ -1,39 +1,74 @@
+#
+# Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
+# Copyright (c) 2007-2018 Andreas Schneider <asn@cryptomilk.org>
+# Copyright (c) 2018      Anderson Toshiyuki Sasaki <ansasaki@redhat.com>
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+#
+# Modifed version from https://github.com/xbmc/libssh/blob/667fb5f9a9c96f210583dbfb11755c43250c5e55/cmake/Modules/AddCMockaTest.cmake
+#.rst:
+# AddTestCMocka
```

Comment:
Why did we rename the file originally? Keep it matching the source:

Rename the file:
`AddTestCMocka.cmake` -> `AddCMockaTest.cmake`

Reply:
there's no difference. Can we keep this name so we don't make more changes to more files importing this module?
We are already mentioning this is a modified version and nothing stop us from using a different name. Isn't this a nit?